### PR TITLE
[PT FE] Add torchbenchmark models to GHA tests

### PR DESCRIPTION
**Pull request by mvafin**
### Details:
 - *Add torchbenchmark models to GHA tests*

### Tickets:
 - *None*

### CODE REVIEW

File: tests/model_hub_tests/torch_tests/torchbench_models [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+#detectron2_fasterrcnn_r_101_fpn,None
+#detectron2_fasterrcnn_r_50_c4,None
+#detectron2_fasterrcnn_r_50_dc5,None
+#detectron2_fasterrcnn_r_50_fpn,None
+#detectron2_fcos_r_50_fpn,None
+#detectron2_maskrcnn,None
+#detectron2_maskrcnn_r_101_c4,None
+#detectron2_maskrcnn_r_101_fpn,None
+#detectron2_maskrcnn_r_50_c4,None
+#detectron2_maskrcnn_r_50_fpn,None
```
Comment (mvafin) [inline tests/model_hub_tests/torch_tests/torchbench_models:26]:
```suggestion
#detectron2_fasterrcnn_r_101_c4,None - Already tested by det2 tests
#detectron2_fasterrcnn_r_101_dc5,None - Already tested by det2 tests
#detectron2_fasterrcnn_r_101_fpn,None - Already tested by det2 tests
#detectron2_fasterrcnn_r_50_c4,None - Already tested by det2 tests
#detectron2_fasterrcnn_r_50_dc5,None - Already tested by det2 tests
#detectron2_fasterrcnn_r_50_fpn,None - Already tested by det2 tests
#detectron2_fcos_r_50_fpn,None - Already tested by det2 tests
#detectron2_maskrcnn,None - Already tested by det2 tests
#detectron2_maskrcnn_r_101_c4,None - Already tested by det2 tests
#detectron2_maskrcnn_r_101_fpn,None - Already tested by det2 tests
#detectron2_maskrcnn_r_50_c4,None - Already tested by det2 tests
#detectron2_maskrcnn_r_50_fpn,None - Already tested by det2 tests
```

File: tests/model_hub_tests/torch_tests/torchbench_models [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+Super_SloMo,None,xfail,Unsupported ops aten::l1_loss aten::mse_loss
+#alexnet,None - Already tested by torchvision tests
+basic_gnn_edgecnn,None,xfail,Accuracy validation failed
+basic_gnn_gcn,None,xfail,Unsupported ops aten::pow_ aten::scatter_add_
+basic_gnn_gin,None,xfail,Unsupported op aten::scatter_add_
+basic_gnn_sage,None,xfail,Unsupported op aten::scatter_add_
+#cm3leon_generate,None,skip,No install.py is found
+dcgan,None
+demucs,None,xfail,Unsupported op aten::lstm
+#densenet121,None
```
Comment (mvafin) [inline tests/model_hub_tests/torch_tests/torchbench_models:14]:
```suggestion
#densenet121,None - Already tested by torchvision tests
```

File: tests/model_hub_tests/torch_tests/test_torchbench.py [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+            model(*self.example)
+        return model
+
+    def teardown_class(self):
+        # cleanup tmpdir
+        self.repo_dir.cleanup()
+
+    @pytest.mark.parametrize("name", process_pytest_marks(_model_list_path))
+    @pytest.mark.nightly
+    @pytest.mark.precommit  # todo: remove
```
Comment (mvafin) [inline tests/model_hub_tests/torch_tests/test_torchbench.py:58]:
```suggestion
```

File: tests/model_hub_tests/torch_tests/test_torchbench.py [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+        model, self.example = model_cls.get_module()
+        self.inputs = self.example
+        # initialize selected models
+        if model_name in ["BERT_pytorch", "yolov3"]:
+            model(*self.example)
+        return model
+
+    def teardown_class(self):
+        # cleanup tmpdir
+        self.repo_dir.cleanup()
```
Comment (eaidova) [inline tests/model_hub_tests/torch_tests/test_torchbench.py:54]: Is it no need to run common teardown from base class?

Comment (mvafin) [reply tests/model_hub_tests/torch_tests/test_torchbench.py:54]: There is no common teardown class method, but yeah would need to add in later PRs

**Pull request merged**